### PR TITLE
Support phpunit 9

### PR DESCRIPTION
**Pull request by pl-github**
- Updated Constraint classes with "?bool" return type to be compatible with PHPUnit 9 and added missing `return null`
- Updated travisCI config to run tests with PHP 7.3 and PHP 7.4
- added PHPUnit 9 to composer.json
- updated Changelog and README

### CODE REVIEW

**Comment by emodric** [comment]
Hi,

Can this be merged and tagged soon?

Thanks!

**Comment by pl-github** [comment]
@Nyholm  Is there anything else I can do to prepare the merge request for a faster processing? Undo the changes in the changelog or the readme? Introduce a major version jump?

**Comment by chq81** [comment]
@Nyholm I would be very happy if this could be merged soon, as I would like to use that with symfony 5 and phpunit 9. Thanks in advance.

**Comment by emodric** [comment]
Hi! Any news on this?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.1.1
```
Comment (Nyholm) [inline CHANGELOG.md:3]: It is a new feature. 
```suggestion
## 4.2.0
```

**Comment by Nyholm** [comment]
Thank you for this PR. 

I've tagged a release: https://github.com/SymfonyTest/SymfonyConfigTest/releases/tag/4.2.0

**Comment by emodric** [comment]
Thank you @Nyholm !

**Pull request merged**